py/trtorch/ptq.py

```diff
@@ -0,0 +1,61 @@
+from typing import List, Dict, Any
+import torch
+import os
+
+import trtorch._C
+from trtorch._compile_spec import _parse_compile_spec
+from trtorch._version import __version__
+from types import FunctionType
+import tensorrt as trt

```

Comment:
We should not be importing tensorrt python api. Any thing we need from here should be reexported from our bindings.

Reply:
Fixed it. Cleaned it up.